ui_testing/pages/base_selenium.py

```diff
@@ -704,8 +704,9 @@ def get_row_cells_elements_related_to_header(self, row, table_element='general:t
         cells_elements = {}
         headers = self.get_table_head_elements(element=table_element)
         headers_text = [header.text for header in headers]
+        time.sleep(5)
+        import ipdb;ipdb.set_trace()	
```

Comment:
when I edit title to Ms I got error in this line that :
'IndexError: list index out of range'